backend/app/core/sessions.py

```diff
@@ -28,23 +27,31 @@ def __getitem__(self, key: str):
         return raw and pickle.loads(raw)
 
     def __setitem__(self, key: str, value: Any):
-        expireTime = timedelta(hours=1)
         self.client.set(
             key,
             pickle.dumps(value, protocol=pickle.HIGHEST_PROTOCOL),
-            ex=settings.server_sessions.expires_in,
         )
 
     def __delitem__(self, key: str):
         self.client.delete(key)
 
+    def set_item_with_expiry_time(self, key: str, value: Any, expires_in: int = None):
+        self.client.set(
+            key,
+            pickle.dumps(value, protocol=pickle.HIGHEST_PROTOCOL),
+            ex=expires_in,
+        )
+
     def generate_session_id(self) -> str:
         sessionId = generate_id()
         while self.client.exists(sessionId):
             sessionId = generate_id()
         return sessionId
 
 
+redis_session_storage = RedisSessionStorage()
```

Comment:
I would make this variable just "whatever session storage we are currently using" so if we grow support for more than one storage backend we only have to change this here.
```suggestion
session_storage = RedisSessionStorage()
```

Reply:
Done.